finetuning-llama-7b-on-l4/README.md

```diff
@@ -0,0 +1,387 @@
+# Tutorial: Finetuning Llama 7b on GKE using L4 GPUs
+
+Large language models (LLMs) have achieved remarkable progress in recent years from answering your questions in an informative way to generating code. However, achieving accuracy in LLMs is still a challenging task, especially in a cost-effective way. We are proud to offer high-performance, cost-efficient, and scalable AI inference offerings. 
```

Comment:
The last sentence in this paragraph makes a sudden pivot from discussing the challenges of achieving high accuracy when training LLMs, to our inference offering, then in the following paragraph returns back to fine-tuning. I think we should remove the sentence introducing our inference offerings, since this tutorial is about fine-tuning and that sentence is placed such that it sounds like it's introducing inference as the topic of the tutorial.

Reply:
I'm not sure who added this paragraph but I think we should remove this whole paragraph. note I copied this over from a blog post I've been working on